src/plugins/data_views/server/fetcher/index_patterns_fetcher.ts

```diff
@@ -64,18 +64,12 @@ export class IndexPatternsFetcher {
     fields?: string[];
   }): Promise<{ fields: FieldDescriptor[]; indices: string[] }> {
     const { pattern, metaFields = [], fieldCapsOptions, type, rollupIndex, indexFilter } = options;
-    const patternList = Array.isArray(pattern) ? pattern : pattern.split(',');
     const allowNoIndices = fieldCapsOptions
       ? fieldCapsOptions.allow_no_indices
       : this.allowNoIndices;
-    let patternListActive: string[] = patternList;
-    // if only one pattern, don't bother with validation. We let getFieldCapabilities fail if the single pattern is bad regardless
-    if (patternList.length > 1 && !allowNoIndices) {
-      patternListActive = await this.validatePatternListActive(patternList);
-    }
```

Comment:
For me this looks redundant if we decide no longer to show the message for existing data views without matching indices. Seems it was introduced in https://github.com/elastic/kibana/pull/90170?